### DISCUSSION TITLE
Add an option to choose which feeds will load the full content

### DISCUSSION THREAD

**Discussion by goliv04053** [jocmp/capyreader#284] [Feature Requests]
This feature would apply to specific feeds, allowing users to manually control whether a feed loads its full content.

Example: https://www.web3isgoinggreat.com/feed.xml

This feed only loads with full content enabled to each article. But there are some feeds which I do not use load full content.

**Comment by jocmp** [comment]
This is what the "Sticky Full Content" option in settings should do. I've attached a video of what should happen: when I navigate to the first article and click the Full Content button it remembers it for all other articles in that feed. The option does not apply to the article in The Verge feed unless you toggle it on. Let me know if this is what you mean or if you're thinking of something else.

https://github.com/user-attachments/assets/ec316f94-c792-4e35-91d8-c6bf1283e9bf
